docs/en/compute/HBase.md

```diff
@@ -30,29 +30,58 @@ Therefore, the configuration of Alluxio is done mostly in HBase configuration fi
 
 ### Set property in `hbase-site.xml`
 
-Change the `hbase.rootdir` property in `conf/hbase-site.xml`:
-> You do not need to create the `/hbase` directory in Alluxio, HBase will do this for you.
+Set the following properties in `conf/hbase-site.xml` and make sure all HBase cluster nodes
+have the configuration.
 
+Set the `hbase.rootdir` property as follows: 
 ```xml
 <property>
   <name>hbase.rootdir</name>
   <value>alluxio://master_hostname:port/hbase</value>
 </property>
 ```
 
-Add the following property to the same file `hbase-site.xml`.
-(make sure it is configured in all HBase cluster nodes):
+> You do not need to create the `/hbase` directory in Alluxio, HBase will do this for you.
+
+You also need to add the FS implementation classes to HBase configuration. These classes are provided in Alluxio Client jar.
+
+```xml
+<property>
+  <name>fs.alluxio.impl</name>
+  <value>alluxio.hadoop.FileSystem</value>
+</property>
+<property>
+  <name>fs.alluxio-ft.impl</name>
+  <value>alluxio.hadoop.FaultTolerantFileSystem</value>
+</property>
+<property>
+  <name>fs.AbstractFileSystem.alluxio.impl</name>
+  <value>alluxio.hadoop.AlluxioFileSystem</value>
+</property>
+```
```

Comment:
FS implementation classes added here. I'm not sure if this is compulsory but all docs indicate that they are needed.